README.md

```diff
@@ -28,22 +28,29 @@ default version after a deployment.
 
 - [x] Registration of new worker versions
 - [x] Creation of versioned worker deployment resources
-- [x] Deletion of unreachable worker deployments
+- [x] Deletion of drained worker deployments
 - [x] Manual, Blue/Green, and Progressive rollouts of new worker versions
 - [ ] Autoscaling of worker deployments
 - [ ] Automated rollover to compatible worker versions
 - [ ] Canary analysis of new worker versions
```

Comment:
is this bullet point referring to analysis beyond just checking that the canary workflow completed successfully?